### PR TITLE
feat(website): add wallet landing page

### PR DESCRIPTION
**Pull request by hadzhehsen**

### CODE REVIEW

File: apps/website/pages/wallet.tsx [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+              <LandingButton store="apple" />
+              <Link
+                href="https://play.google.com/store/apps/details?id=com.haqq.wallet"
+                target="_blank"
+              >
+                <LandingButton store="google" isAvailable />
+              </Link>
+            </div>
+          </div>
+          <div className="flex justify-center">
```
Comment (olegshilov) [inline apps/website/pages/wallet.tsx]:
```suggestion
          <div className="flex flex-1 justify-center">
```

**Review by olegshilov — CHANGES_REQUESTED** [review]

File: apps/website/lib/fonts.ts [hunk 2]
```diff
@@ -1,4 +1,6 @@
 import localFont from '@next/font/local';
+import { El_Messiri } from '@next/font/google';
+import { Manrope } from '@next/font/google';
```
Comment (olegshilov) [inline apps/website/lib/fonts.ts]:
```suggestion
import { El_Messiri, Manrope } from '@next/font/google';
```

File: apps/website/pages/wallet.tsx [hunk 3]
```diff
@@ -0,0 +1,384 @@
...
+        'flex flex-col items-center justify-start relative flex-1',
+      )}
+    >
+      <Image
+        alt=""
+        src={walletImageData.src}
+        fill
+        className="z-[-1] object-cover bg-center"
+      />
+      <header className="header h-[70px] w-full py-[20px] sm:py-[22px]">
```
Comment (olegshilov) [inline apps/website/pages/wallet.tsx]:
```suggestion
      <header className="h-[70px] w-full py-[20px] sm:py-[22px]">
```

File: apps/website/pages/wallet.tsx [hunk 4]
```diff
@@ -0,0 +1,384 @@
...
+            <Image
+              src={iphoneWalletScreenshotData.src}
+              alt="HaqqWallet"
+              width={iphoneWalletScreenshotData.width}
+              height={iphoneWalletScreenshotData.height}
+            />
+          </div>
+        </div>
+      </div>
+      <footer className="footer h-[96px] w-full">
```
Comment (olegshilov) [inline apps/website/pages/wallet.tsx]:
```suggestion
      <footer className="h-[96px] w-full">
```

**Review by olegshilov — CHANGES_REQUESTED** [review]

File: apps/website/lib/fonts.ts (hunk 2 above)
Comment (hadzhehsen) [reply apps/website/lib/fonts.ts]: Done

File: apps/website/pages/wallet.tsx (hunk 3 above)
Comment (hadzhehsen) [reply apps/website/pages/wallet.tsx]: Done

File: apps/website/pages/wallet.tsx (hunk 4 above)
Comment (hadzhehsen) [reply apps/website/pages/wallet.tsx]: Done

File: libs/website/ui-kit/src/lib/icons/discord-icon.tsx [hunk 5]
```diff
@@ -4,12 +4,12 @@ export function DiscordIcon({ className }: { className?: string }) {
       width="26"
       height="26"
       viewBox="0 0 27 26"
-      fill="none"
+      fill="currentColor"
```
Comment (olegshilov) [inline libs/website/ui-kit/src/lib/icons/discord-icon.tsx]: revert

File: libs/website/ui-kit/src/lib/icons/link-arrow-icon.tsx [hunk 6]
```diff
@@ -3,8 +3,7 @@ export function LinkArrowIcon() {
     <svg
       className="hidden sm:block sm:invisible group-hover:visible h-[24px] w-[24px]"
       viewBox="0 0 25 24"
-      fill="none"
-      xmlns="http://www.w3.org/2000/svg"
+      fill="currentColor"
```
Comment (olegshilov) [inline libs/website/ui-kit/src/lib/icons/link-arrow-icon.tsx]: revert

File: libs/website/ui-kit/src/lib/icons/medium-icon.tsx [hunk 7]
```diff
@@ -4,12 +4,12 @@ export function MediumIcon({ className }: { className?: string }) {
       width="26"
       height="26"
       viewBox="0 0 27 26"
-      fill="none"
+      fill="currentColor"
```
Comment (olegshilov) [inline libs/website/ui-kit/src/lib/icons/medium-icon.tsx]: revert

File: libs/website/ui-kit/src/lib/icons/telegram-icon.tsx [hunk 8]
```diff
@@ -4,12 +4,12 @@ export function TelegramIcon({ className }: { className?: string }) {
       width="26"
       height="26"
       viewBox="0 0 26 26"
-      fill="none"
+      fill="currentColor"
```
Comment (olegshilov) [inline libs/website/ui-kit/src/lib/icons/telegram-icon.tsx]: revert

File: libs/website/ui-kit/src/lib/icons/twitter-icon.tsx [hunk 9]
```diff
@@ -4,12 +4,12 @@ export function TwitterIcon({ className }: { className?: string }) {
       width="26"
       height="26"
       viewBox="0 0 26 26"
-      fill="none"
+      fill="currentColor"
```
Comment (olegshilov) [inline libs/website/ui-kit/src/lib/icons/twitter-icon.tsx]: revert

File: libs/website/ui-kit/src/lib/icons/youtube-icon.tsx [hunk 10]
```diff
@@ -4,12 +4,12 @@ export function YoutubeIcon({ className }: { className?: string }) {
       width="26"
       height="26"
       viewBox="0 0 27 26"
-      fill="none"
+      fill="currentColor"
```
Comment (olegshilov) [inline libs/website/ui-kit/src/lib/icons/youtube-icon.tsx]: revert

**Review by olegshilov — CHANGES_REQUESTED** [review]

File: libs/website/ui-kit/src/lib/icons/discord-icon.tsx (hunk 5 above)
Comment (hadzhehsen) [reply libs/website/ui-kit/src/lib/icons/discord-icon.tsx]: Done

File: libs/website/ui-kit/src/lib/icons/link-arrow-icon.tsx (hunk 6 above)
Comment (hadzhehsen) [reply libs/website/ui-kit/src/lib/icons/link-arrow-icon.tsx]: Done

File: libs/website/ui-kit/src/lib/icons/telegram-icon.tsx (hunk 8 above)
Comment (hadzhehsen) [reply libs/website/ui-kit/src/lib/icons/telegram-icon.tsx]: Done

File: libs/website/ui-kit/src/lib/icons/youtube-icon.tsx (hunk 10 above)
Comment (hadzhehsen) [reply libs/website/ui-kit/src/lib/icons/youtube-icon.tsx]: Done

**Pull request merged**